js/profile-w3c-common.js

```diff
@@ -49,10 +49,11 @@ define(
     "core/best-practices",
     "core/figures",
     "core/webidl",
+    "core/link-to-dfn-internal",
```

Comment:
Unfortunately, I don’t think we can’t rename this, as it would break other profiles.

Reply:
Alternatives? We do need to separate out linking of internal and external definitions, as I believe.

Reply:
doing the separation is fine, but this module should remain as is (and retain backwards compatibility). The new external xrefs can just live in “*-external.js”

Reply:
> doing the separation is fine, but this module should remain as is

Didn't quite get that. "remain same" in what sense?

I actually want to separate it into two parts as:
- I do not want `link-to-dfn` to emit `"warn", Linking error: not matching <dfn>` while it has just looked for internal dfns

Reply:
That’s ok, but that shouldn’t require the module to be renamed. The external xref module could be imported by this module, for instance. It doesn’t necessarily also need to be a separate plugin.

Reply:
> Didn't quite get that. "remain same" in what sense?

To be clear, in the sense that the external reference code should be a progressive enhancement, and not a breaking change.

Reply:
I’ll try to be online in about 2.5 hours so we can discuss... still 😴 🛏  here (yay jet lag!).